### PR TITLE
Homework 22

### PR DESCRIPTION
**Pull request by OlexiyVasylenkoDev**

### CODE REVIEW

File: src/accounts/models.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    user = models.ForeignKey(to="accounts.CustomUser", on_delete=models.CASCADE, blank=True, null=True)
+
+    class Meta:
+        abstract = True
+
+
+class SingleVolunteerProfile(CustomProfile):
+    first_name = models.CharField(_("first name"), max_length=150, blank=False, null=False, default=None)
+    last_name = models.CharField(_("last name"), max_length=150, blank=False, null=False, default=None)
+    opportunity = models.ManyToManyField(to="volunteering.Opportunity")
```
Comment (mikolaz27) [inline src/accounts/models.py]: move this relation to opportunity

File: src/accounts/views.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+from django.shortcuts import render # NOQA
+
+# Create your views here.
+# from django.urls import reverse_lazy
+# from django.views.generic import CreateView, TemplateView
+#
+# from accounts.forms import RegistrationForm
+#
+#
+# class IndexView(TemplateView):
```
Comment (mikolaz27) [inline src/accounts/views.py]: remove

File: src/accounts/models.py (hunk 1 above)
Comment (OlexiyVasylenkoDev) [reply src/accounts/models.py]: I have this field in VolunteersOrganisation model (as well as with need field in another two profiles). Should I move both of them to opportunity as two different fields like this?

SingleVolunteerProfile = models.ManyToManyField(to="accounts.SingleVolunteerProfile")
VolunteersOrganisationProfile = models.ManyToManyField(to="accounts.VolunteersOrganisationProfile ")

File: src/accounts/views.py (hunk 2 above)
Comment (OlexiyVasylenkoDev) [reply src/accounts/views.py]: Done

**Pull request merged**